tools/pairing-support-tool/README.md

```diff
@@ -0,0 +1,53 @@
+
+### Is this project for me?
+**YES**. Anyone, independently of their area of expertise or previous knowledge, can contribute. Keep reading to learn more and write us if you have questions!
+
+## The high level plan
+Everything, including the plan, will be taken care off by the project team. Just to give you and idea, this is what we have in mind:
+
+1. Setting up a team: below in this article you'll find a draft of the minimum roles we think this team should have. If you want to take any of those or propose some additional role let us know when you write us.
+1. Kick of the project: once the team is set, we'll announce it and start the activities! 🦄
+1. Define the processes the team will follow
+1. Gather/refine requirements
+1. Develop the pieces of the system
+1. Test them
+1. Repeat. When there is something that can be used in a workshop already, let's do it! And then repeat again :)
+
+## What should the pairing tool do?
+### Suggested initial requirements
+Just as food for thoughts, this is a list of what we have in mind. How? That the team will decide :)
+
+* Match students and coaches based on the technologies they want to learn or can teach
+* Produce a report/screen where the organisers can see the resulting pairs and announce them after the round of presentations.
+* Develop the tool in a way that it could later be integrated with the codebar site (so it can be used in other chapters as well and  could share the Database)
+* Keep record of who was paired with whom each workshop. Use it for next workshops?
+
+## Suggested team
+As said before, there is space for creativity related to how to set up the team. **These roles are perfect place for anyone willing to try something new**. You don't need to be an expert in the role to take it. There will be mentors helping you during the way.
+
+This list is just to inspire participants. As said, if you want to contribute we'll find you a role to take:
+* Designers
+* Developers (front-end and back-end. Also SQL fans)
+* DevOps / System administrators
+* Product owners / Product managers
+* Project leaders
+* Testers
+* UX passionates
+* Mentors/coaches for all these profiles
+
+## Pre-requisites
+The only pre-requisite we would like to mention is **availability**. We want this project to reach something usable around the end of the year, so we'll have to set up a steady pace of work during the next 4/5 months. We estimate that anyone contributing should have between 4 and 8 hours to dedicate to the project per week. Of course this will depend on which role you choose, but more or less as an estimate this is what we foresee.
+
+## Want to be part of this project?
+
+Write us an email to barcelona@codebar.io. Indicate which role you would like to take or mentor.
```

Comment:
instead of a mail, why not use github? They can comment on a issue to show their interest. It will be easier afterwards to have the github profile to handle the permission on the github repo.

Reply:
Sound good! I've created an issue and liked it already.